scripts/pipeline/slack_notifications/templates/post_environment_domains.j2

```diff
@@ -7,10 +7,24 @@
             "text":
             {
                 "type": "plain_text",
-                "text": "Online LPA Development Environment Ready",
+                "text": "Online LPA Development Environment Deployed",
                 "emoji": false
             }
-        }
+        },
+		{
```

Comment:
Indentation is inconsistent here.

Reply:
i'll take a look :)

Reply:
yep  - it's placement of braces - another one of those potential things for code wars 🤣 . i'll fix and push - if you could re-approve please?